development-strategy.md

```diff
@@ -6,50 +6,91 @@
 ![Wireframe Indonesian Time](images/wireframe_ID_radoslav_unsplash.jpg)
 
 ## 0. Initial
-- ReadME
-- Development Strategy
+- Create a ReadME file
+- Create Development Strategy MD
 
 **SetUp**
 - Create master boiler index html
 - Create branches per user story (4 branches)
 - Add assest for each user story
 
-## 1. User Story 1: Display of the Clock
-A user can see a page designed to show the local time (Belgium) with a digital clock
+## 1. User Story 1: Display of the Header (Homepage)
+As a user; I want to see a well styled webpage.
 
 DOM
-- Create header `<h1>` containing the name of local country
-- Create the format for digital clock
+- Create a parent Div.  
+- Create the buttons on the homepage
```

Comment:
sorry, what do you mean the buttons on the homepage? are they the country buttons?